src/ui/components/filters/filteroptionscomponent.js

```diff
@@ -134,20 +148,27 @@ class FilterOptionsConfig {
 
   /**
    * If no previous options were stored in persistentStorage, default to options marked
-   * as appliedOnLoad.
+   * as selected. If multiple options are marked as selected for 'singleoption', only the
+   * first should be selected.
    * @param {*} options
+   * @returns {Array<Object>}
    */
-  setAppliedOnLoad (options) {
+  setSelected (options) {
```

Comment:
Can we consolidate setPreviousOptions and setSelected into one, single setSelected method? I think it could help reduce a bit of complexity if we had a single method whose responsibility was setting the selected attribute for an option(s). As it stands, we essentially have two different methods for doing this.

Reply:
updated!